intelmq/bots/BOTS

```diff
@@ -347,6 +347,11 @@
             "module": "intelmq.bots.parsers.bitcash.parser",
             "parameters": {}
         },
+	"BotScout": {
```

Comment:
And what's that now?

Reply:
Lol, test for new parser i am working on....let me remove it.  sorry, not very good with git branches yet.